Java/src/com/company/Objects.java

```diff
@@ -0,0 +1,45 @@
+package com.company;
+
+public class Objects {

```

Comment:
You'd better rename this class to `ObjectDemo` and make your object itself a static class in your demo, or make it separate at all.